broker/src/main/java/org/apache/rocketmq/broker/plugin/MessageStoreFactory.java

```diff
@@ -22,7 +22,7 @@
 import org.apache.rocketmq.store.MessageStore;
 
 public final class MessageStoreFactory {
-    public final static MessageStore build(MessageStorePluginContext context, MessageStore messageStore)
+    public static MessageStore build(MessageStorePluginContext context, MessageStore messageStore)

```

Comment:
It seems this change has nothing to do with the issue, you may open another issue to change it.